amplify.yml

```diff
@@ -8,6 +8,7 @@ frontend:
       commands:
         - rm -rf .next/
         - cp .env.$USER_BRANCH .env
+        - echo "PLAID_SECRET=$PLAID_SECRET" >> .env
```

Comment:
PLAID_SECRET is an environment variable in AWS amplify. Tried with the secrets feature but has no support to fetch secrets locally unless you are running an amplify sandbox which means extra costs when developing